modules/core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/DocumentBlobManagerComponent.java

```diff
@@ -295,7 +265,7 @@ public void updateBlob(Document doc, Consumer<BlobUpdateContext> contextFiller)
         }
         BlobProvider blobProvider = Framework.getService(BlobManager.class).getBlobProvider(blob);
         if (blobProvider == null) {
-            log.error("No blob provider found for blob: " + blob.getKey());
+            log.error("No blob provider found for blob: {}", blob.getKey());
```

Comment:
`blob::getKey`
(the perf aspect is trivial, but you may get warning from some code quality tools if you don't use constants (or a method handle or a lambda) in log statements)